force_wfmanager/central_pane/analysis.py

```diff
@@ -1,9 +1,52 @@
 from pyface.tasks.api import TraitsTaskPane
-from traitsui.api import View
+
+from traits.api import List, Instance, Int, Any, Str, HasStrictTraits
+
+from traitsui.api import View, HGroup, VGroup, Spring
+
+
+class Result(HasStrictTraits):
+    name = Str()
+    value = Any()
+
+
+class EvaluationStep(HasStrictTraits):
+    _step = 0
+
+    @classmethod
+    def init_step_count(cls):
```

Comment:
This method will be useful when restarting the computation, you want to clear the result panes and restart the count of evaluation steps. I don't understand your comment about not needing the model, it is exactly the point of this PR to create the model.